src/Options.php

```diff
@@ -705,6 +727,7 @@ private function configureOptions(OptionsResolver $resolver): void
             'max_value_length' => 1024,
             'http_proxy' => null,
             'capture_silenced_errors' => false,
+            'type_serializers' => [],
```

Comment:
Since all of this seems to be targeting just objects, I would prefer renaming it `class_serializers`

Reply:
That is a better word for it indeed!